yt/visualization/profile_plotter.py

```diff
@@ -704,14 +701,8 @@ def _set_font_properties(self):
 
     def _get_field_log(self, field_y, profile):
         yfi = profile.field_info[field_y]
-        if self.x_log is None:
-            x_log = profile.x_log
-        else:
-            x_log = self.x_log
-        if field_y in self.y_log:
-            y_log = self.y_log[field_y]
-        else:
-            y_log = yfi.take_log
+        x_log = self.x_log if self.x_log is not None else profile.x_log
```

Comment:
I think this isn't exactly readable. Would you mind splitting the logic for x_log across a couple of lines here?

Reply:
you got it